test/RecipeBot.Domain.TestUtils/RecipeDomainModelTestFactory.cs

```diff
@@ -98,22 +108,28 @@ public RecipeModel CreateWithImageAndFields()
     {
         return CreateRecipeModel(new[]
         {
-            CreateRecipeFiledModel(1),
-            CreateRecipeFiledModel(2),
-            CreateRecipeFiledModel(3)
+            CreateRecipeFieldModel(1),
+            CreateRecipeFieldModel(2),
+            CreateRecipeFieldModel(3)
         }, "https://recipeBot.recipe.image");
     }
 
+    private RecipeModel CreateRecipeModel(RecipeCategory recipeCategory)
+    {
+        string title = GetStringWithRandomLength('x', maxTitleLength);
+        return new RecipeModel(CreateAuthorModel(), recipeCategory, Enumerable.Empty<RecipeFieldModel>(), title);
+    }
+
```

Comment:
Check if this overload is really necessary or whether it is enough to inline this